app/scenes/Invite.tsx

```diff
@@ -140,19 +140,23 @@ function Invite({ onSubmit }: Props) {
   const collectionAccessNote = (
     <span>
       <Trans>Invited members will receive access to</Trans>{" "}
-      <Tooltip
-        content={
-          <>
-            {collections.nonPrivate.map((collection) => (
-              <li key={collection.id}>{collection.name}</li>
-            ))}
-          </>
-        }
-      >
-        <Def>
-          <Trans>{{ collectionCount }} collections</Trans>
-        </Def>
-      </Tooltip>
+      {collectionCount > 0 ? (
+        <Tooltip
+          content={
```

Comment:
It's probably fine to just return `undefined` here in this scenario and suppress the tooltip as a simple fix